CONTRIBUTING.md

```diff
@@ -48,6 +48,58 @@ python run-clang-format.py \
 
 where `$CLANG_FORMAT` denotes the path to the downloaded binary.
 
+## Adding Third Party Libraries
+
+The following steps outline how to add third party libraries to torchtext:
+
+1. Add the git repo you care about as a submodule. Here is a great
```

Comment:
Not a native English speaker but I feel something about the expression `care about`.

Reply:
Good callout. Let me remove this